soc/nxp/imx/imx8m/Kconfig.soc

```diff
@@ -0,0 +1,34 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+config SOC_SERIES_IMX_8M
+	bool
+	select SOC_FAMILY_NXP_IMX
+
+config SOC_SERIES
+	string
+	default "imx8m" if SOC_SERIES_IMX_8M
```

Comment:
as above

Reply:
done - removed `string` and updated the name to `SOC_SERIES_IMX8M`